test/docs/testplans/vnet_to_vnet.md

```diff
@@ -0,0 +1,54 @@
+- Data traffic will be sent from VNET to VNET. In particular, it allows the following features: VM to VM communication in VNET, route support, LPM support, ACL support. This is to verify the following performance properties: CPS, flow, PPS, and rule scale.
+- Depending on the test cases, a combination of UDP traffic for IPv4, IPv6 addresses will be generated
+
+### **Test Cases**
+[TODO: Update Table]
+
+| Test Description                                             | Expected Result               |
+| :----------------------------------------------------------- | :------------------------ |
+| UDP IPv4 inbound  | Traffic originating from one VNet simulated terminates on another simulated VNet. Counters in DUT in the inbound direction are accurately incremented. <br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br> Verify configured throughput rate is achieved in traffic statistics  |
+| UDP IPv4 outbound | Traffic originating from one VNet simulated terminates on another simulated VNet. Counters in DUT in the outbound direction are accurately incremented. <br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br> Verify configured throughput rate is achieved in traffic statistics  |
+| UDP IPv4 bidirectional  | Traffic originating from one VNet simulated terminates on another simulated VNet. Verify bidrectional traffic Counters in DUT in the outbound direction are accurately incremented. <br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br> Verify configured throughput rate is achieved in traffic statistics  |
+| UDP IPv6 inbound | Traffic originating from one VNet simulated terminates on another simulated VNet. Counters in DUT in the inbound direction are accurately incremented. <br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br> Verify configured throughput rate is achieved in traffic statistics  |
+| UDP IPv6 outbound direction | Traffic originating from one VNet simulated terminates on another simulated VNet. Counters in DUT in the outbound direction are accurately incremented. <br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br> Verify configured throughput rate is achieved in traffic statistics  |
+| UDP IPv6 bidirectional  | Traffic originating from one VNet simulated terminates on another simulated VNet. Verify bidrectional traffic Counters in DUT in the outbound direction are accurately incremented. <br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br> Verify configured throughput rate is achieved in traffic statistics  |                                                 
+|  ACL inbound | Verify data traffic is allowed and unwanted traffic based on ACL is dropped. Verify ACL inbound counters on DUT are incremented
```

Comment:
Are we testing both IPv4 and IPv6 ACLs?

Reply:
yes, i believe so. I can check with desai and update here.